src/test/java/nl/tudelft/ewi/devhub/webtests/IssuesTest.java

```diff
@@ -186,6 +198,15 @@ public void testAddRemoveLabel(){
 		assertEquals(0x00ccff, label.getColor());
 	}
 	
+	@After
+	public void deleteIssues(){
+		issues.findAllIssues(group.getRepository()).forEach(x -> {
+			Stream<? extends IssueComment> comments = issueComments.getMostRecentIssueComments(Lists.<GroupRepository>asList(group.getRepository(), new GroupRepository[0]),10);
+			comments.forEach(comment -> issueComments.delete(comment));
+			issues.delete(x);
```

Comment:
This can be done differently. Enable `orphanRemoval = true` and `CascadeType.DELETE` on `comments` in `AbstractIssue`. Then you can just do: `issues.findAllIssues(group.getRepository()).forEach(issues::delete)`.